xmss_commons.c

```diff
@@ -122,6 +122,27 @@ void gen_leaf_wots(const xmss_params *params, unsigned char *leaf,
     l_tree(params, leaf, pk, pub_seed, ltree_addr);
 }
 
+void gen_leaf_wots_mp(const xmss_params *params, unsigned char *leaf,
+                   const unsigned char *sk_seed, const unsigned char *pub_seed,
+                   uint32_t ltree_addr[8], uint32_t ots_addr[8])
+{
+    unsigned char seed[params->n];
+    unsigned char pk[params->wots_sig_bytes];
+    #ifdef FORWARD_SECURE
+    unsigned int i;
+
+    // in a forward secure setting, the sk_seed already contains the current seed
```

Comment:
Nit: Start the comment with a capital letter.

Reply:
fix in https://github.com/kj3moraes/further-testing-xmss/pull/5/commits/107a02b24827f62daab30daf264ca5987c75cc4b